enhancements/installer/assisted-installer.md

```diff
@@ -0,0 +1,349 @@
+---
+title: assisted-installer
+authors:
+  - "@avishayt"
+  - "@hardys"
+  - "@dhellmann"
+reviewers:
+  - "@beekhof"
+  - "@deads2k"
+  - "@hexfusion"
+  - "@mhrivnak"
+approvers:
+  - "@crawford"
+  - "@abhinavdahiya"
+  - "@eparis"
+creation-date: 2020-06-09
+last-updated: 2020-06-10
+status: implementable
+see-also:
+  - "/enhancements/baremetal/minimise-baremetal-footprint.md"
+---
+
+# Assisted Installer
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
+
+1. Where do the UI and assisted installer application run for
+   disconnected installations?
+
+   * It could run from a host booted with the live ISO. If that host
+     is the bootstrap host, then access to the UI and API would be
+     interrupted when the host reboots to join the cluster. If that
+     host is not part of the cluster initially then the minimum
+     hardware footprint for creating a cluster increases.
+   * It could run on a user's desktop system, either from the ISO,
+     container image, or as a standalone binary. This may assume the
+     user is either running Linux or is willing to reboot their
+     desktop into a live ISO.
+
+2. In a disconnected deployment, how do the agents and assistant
+   application find each other?
+
+   * We could use generic or cluster-specific MDNS broadcast. MDNS is
+     not available in all environments and does not support hosts
+     running on different L2 networks. That also implies the
+     connection direction would be different for disconnected than
+     connected, where the agent reaches up to the service and the
+     service may not be able to reach the agent.
+   * We could build a URL into the ISO. That implies a separate ISO
```

Comment:
Doesnt it make sense to use the same flow - login into the iso UI, generate discovery iso (will bake the url during the generation), and so on